cns/nmagentclient/nmagentclient.go

```diff
@@ -86,3 +96,42 @@ func GetNetworkContainerVersion(
 		networkContainerID, response, err)
 	return response, err
 }
+
+// GetNmAgentSupportedApis :- Retrieves Supported Apis from NMAgent
+func GetNmAgentSupportedApis(
+	getNmAgentSupportedApisURL string) ([]string, string) {
```

Comment:
This function should return an error instead of a message. I recommend reading: https://github.com/golang/go/wiki/CodeReviewComments#in-band-errors